src/main/java/Duke.java

```diff
@@ -1,10 +1,104 @@
+import java.util.Scanner;
+
 public class Duke {
+    public String greetUser() {
+        return "Hello! I'm Duke" + "\n" + "What can I do for you?";
+    }
+
+    public String farewellUser() {
+        return "Bye. Hope to see you again soon!";
+    }
+
     public static void main(String[] args) {
-        String logo = " ____        _        \n"
-                + "|  _ \\ _   _| | _____ \n"
-                + "| | | | | | | |/ / _ \\\n"
-                + "| |_| | |_| |   <  __/\n"
-                + "|____/ \\__,_|_|\\_\\___|\n";
-        System.out.println("Hello from\n" + logo);
+        Duke duke = new Duke();
+        TaskList tl = new TaskList();
+        String greeting = duke.greetUser();
+        System.out.println(greeting);
+        Scanner sc = new Scanner(System.in);
+        String userInput = sc.nextLine();
+
+        while(!userInput.equals("bye")) {
+            try {
+                if (!userInput.equals("list")) {
```

Comment:
This sequence of `if-else` blocks is hard to read and understand. A `switch` case based off the first word of the command would be neater.